test/api_test.js

```diff
@@ -79,6 +79,16 @@ describe("KintoClient", () => {
         .eql(requestMode);
     });
 
+    it("should keep the default timeout in the child HTTP instance", () => {
+      expect(new KintoClient(sampleRemote).http.timeout).eql(5000);
```

Comment:
A default timeout of 5s seems a little small to me, usually it is more something around 30s isn't it?